.github/workflows/runtime-php-testing.yml

```diff
@@ -0,0 +1,44 @@
+name: Runtime testing PHP models
+on:
+  push:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - 'src/generators/php/**'
+      - 'test/runtime/runtime-php/**'
+      - 'test/runtime/**php**'
+
+jobs:
+  runtime-php-test:
+    name: Runtime testing PHP ${{ matrix.php }} Models
+    if: "github.event.pull_request.draft == false &&!((github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'ci: update global workflows')) || (github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'chore(release):')) || (github.actor == 'allcontributors' && startsWith(github.event.pull_request.title, 'docs: add')))"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - 8.0
+          - 8.1
+          - 8.2
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Build library
+        run: npm install && npm run build:prod
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@2.25.2
+        with:
+          php-version: ${{ matrix.php }}
+
+      - name: Generate Models
+        run: npm run generate:runtime:php
```

Comment:
@jonaslagoni  isn't that part missing in Java and Rust pipelines?

Reply:
Java generates it when the project is build: https://github.com/asyncapi/modelina/blob/c0e0268996126ca65caeaa6d4c3d4fbcef1767bc/test/runtime/runtime-java/pom.xml#L119

I think Rust does the same